src/extensions/components/AppDisabledInfo.tsx

```diff
@@ -8,7 +8,15 @@ export const AppDisabledInfo = () => {
   return (
     <Box display="flex" alignItems="center" gap={1} color="default2">
       <DisabledIcon height="10px" width="10px" />
-      <Text size={2} color="default2">
+      <Text
+        size={2}
+        color="default2"
+        display={{
+          desktop: "block",
+          tablet: "block",
+          mobile: "none",
```

Comment:
question: do we have some other way to show that app is disabled on mobile or we just hide that?